hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -132,11 +133,21 @@ spec = around showLogsOnFailure $ do
           withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \node ->
             publishHydraScriptsAs node Faucet
               >>= timedTx tmpDir tracer node
-      it "commits from external wallet" $ \tracer -> do
+      fit "commits from external utxo" $ \tracer -> do
```

Comment:
I would make this test a BehaviorSpec test so that there is less plumbing involved and keep only a very basic test here.

Reply:
same thread as [above](https://github.com/input-output-hk/hydra/pull/903#discussion_r1226646988)
lets continue it there.